funcx_endpoint/tests/funcx_endpoint/executors/high_throughput/test_funcx_manager.py

```diff
@@ -0,0 +1,79 @@
+from funcx_endpoint.executors.high_throughput.funcx_manager import Manager
+from funcx_endpoint.executors.high_throughput.messages import Task
+import queue
+import logging
+import pickle
+import zmq
+import os
+import shutil
+import pytest
+
+
+class TestManager:
+
+    @pytest.fixture(autouse=True)
+    def test_setup_teardown(self):
+        os.makedirs(os.path.join(os.getcwd(), 'mock_uid'))
+        yield
+        shutil.rmtree(os.path.join(os.getcwd(), 'mock_uid'))
+
+    def test_remove_worker_init(self):
+        manager = Manager(logdir='./', uid="mock_uid")
+        manager.worker_map.to_die_count["RAW"] = 0
+        manager.task_queues["RAW"] = queue.Queue()
+
+        manager.remove_worker_init("RAW")
+        task = manager.task_queues["RAW"].get()
+        assert isinstance(task, Task)
+        assert task.task_id == "KILL"
+        assert task.container_id == "RAW"
```

Comment:
Remove some of these assertions to make this test more reliable in the future with other changes. Lines 29 to 31 are probably overkill.

Reply:
need to check both `task_id` and `task_buffer` are `KILL`, which is exactly what is checked on the worker.